### PR TITLE
[TCE] Migrate portugues from static HTML to YAML

### PR DESCRIPTION
**Pull request by mofo37**
# #953 
- Font is incorrect but otherwise looking fine
- https://crimethinc.com/tce/portugues
- https://crimethinc-staging-pr-972.herokuapp.com/tce/portugues

### CODE REVIEW

File: config/locales/to_change_everything/pt.yml [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+        left_text_html: "<p>Quando vemos o que todas as diferentes instituições
+          e os mecanismos de dominação têm em comum, torna-se claro que nossas
+          lutas individuais são também parte de algo maior que nós, algo que pode
+          nos conectar. Quando nos unimos em torno dessa conexão, tudo muda: não
+          apenas nossas lutas, mas também nosso senso de potencial de agir, nossa
+          capacidade de desfrutar, a sensação de que nossas vidas têm sentido.
+          Tudo o que precisamos para nos encontrar umas com as outras é começar
+          a agir de acordo com uma lógica diferente.</p>"
+        right_text_html: |-
+          <h3>Para <br>
```
Comment (astronaut-wannabe) [inline config/locales/to_change_everything/pt.yml]: i think this is a bug in my scraper script.

This `<h3>`  shouldn't be here as it already gets wrapped in an h3 in [`_spread.html.erb`][0]]

```erb
  <div class="rightintro">
    <h3><%= t "to_change_everything.show.#{section}.right_text_html" %></h3>
  </div>
```

[0]:https://github.com/crimethinc/website/blob/master/app/views/to_change_everything/_spread.html.erb#L9

File: config/locales/to_change_everything/pt.yml [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+                    <p>Anarquismo não é um dogma ou um mapa. Não é um sistema que supostamente funcionaria bastando que fosse aplicado corretamente, como a democracia, nem é um objetivo para ser realizado em um futuro muito distante, como o comunismo. É uma forma de agir e nos relacionar que podemos pôr em prática imediatamente. Ele oferece questionamentos que podemos colocar em qualquer situação. Ao invés de questionar se a forma como se desenrola uma ação é legal, ou violenta, ou bondosa, podemos perguntar simplesmente: como ela distribui o poder?</p>
+        third_html: |-
+          <p><span class="blue black">Anarquistas</span> se opõem a toda forma de hierarquia – toda prática que concentra poder na mão de poucas pessoas, todo mecanismo que nos põe distantes de nosso potencial. Contra os sistemas fechados, apreciamos o desconhecido frente a nós, o caos interior através do qual somos capazes de ser livres.</p>
+                    <img src="https://cloudfront.crimethinc.com/tce/images/circle-a-logo-hands-white-large@2x.png">
```
Comment (astronaut-wannabe) [inline config/locales/to_change_everything/pt.yml]: this image is already in the template and being here will cause 2 Ⓐs to be rendered (another bug in the scraper script)

File: app/assets/stylesheets/to_change_everything/_portugues.scss [hunk 3]
```diff
@@ -0,0 +1,511 @@
+.portugues {
+  @import "common/font_styling";
+  /*--- Highlight language in mobile language menu ---*/
+  /*--- TODO: make sure the div selection (e.g. #lang1) matches what was added in
+  app/views/layouts/to_change_everything.html.erb ---*/
+  #lang1  li  > a[href="/tce/portugues"] {
```
Comment (astronaut-wannabe) [inline app/assets/stylesheets/to_change_everything/_portugues.scss]: change this to `#lang2` as portugues is in that div [here][0]

[0]:https://github.com/crimethinc/website/blob/master/app/views/layouts/to_change_everything.html.erb#L188-L197

Comment (veganstraightedge) [reply app/assets/stylesheets/to_change_everything/_portugues.scss]: @astronaut-wannabe If you add both selectors to this in the generator, then this step can be ignored on future locales.

```
  #lang2  li  > a[href="/tce/portugues"],
  #lang1  li  > a[href="/tce/portugues"] {
```

Comment (astronaut-wannabe) [reply app/assets/stylesheets/to_change_everything/_portugues.scss]: @veganstraightedge opened a PR here https://github.com/crimethinc/website/pull/975

File: config/locales/to_change_everything/pt.yml (hunk 2 above)
Comment (astronaut-wannabe) [inline config/locales/to_change_everything/pt.yml]: remove the `` from `third_html` (it is already in `_show.html.erb`

screen shot 2018-12-01 at 3 51 12 pm

**Pull request merged**